jvector-examples/src/main/java/io/github/jbellis/jvector/example/Bench.java

```diff
@@ -24,13 +24,7 @@
 import io.github.jbellis.jvector.example.util.Hdf5Loader;
 import io.github.jbellis.jvector.example.util.ReaderSupplierFactory;
 import io.github.jbellis.jvector.example.util.SiftLoader;
-import io.github.jbellis.jvector.graph.GraphIndex;
-import io.github.jbellis.jvector.graph.GraphIndexBuilder;
-import io.github.jbellis.jvector.graph.GraphSearcher;
-import io.github.jbellis.jvector.graph.ListRandomAccessVectorValues;
-import io.github.jbellis.jvector.graph.NeighborSimilarity;
-import io.github.jbellis.jvector.graph.RandomAccessVectorValues;
-import io.github.jbellis.jvector.graph.SearchResult;
+import io.github.jbellis.jvector.graph.*;
```

Comment:
can we not churn this?

Reply:
Yes, I changed this to a really big number because it bugs me to see it churn between wildcard to not wildcard as requirements change.  Not sure of the best way to have a project-wide code style.